src/components/AdminSettings/MatterbridgeIntegration.vue

```diff
@@ -31,6 +31,13 @@
 		</h2>
 
 		<template v-if="matterbridgeVersion">
+			<p class="settings-hint path-hint">
+				{{ t('spreed', 'You can manually define a custom Matterbridge binary path with occ.') }}
+				<a href="https://github.com/nextcloud/spreed/wiki/Matterbridge-integration" target="_blank" rel="noopener">
```

Comment:
Please put it in docs/ instead and link to https://nextcloud-talk.readthedocs.io/en/latest/